Add features to installation management

julianbollig opened the pull request:
Added a drop-down menu to choose the language and a button to open the dev tools in InstallationManagement. Also slightly changed the style in the Basic tab.

mschuettlerTNG commented on WebUI/src/components/InstallationManagement.vue:
```diff
@@ -98,15 +98,36 @@
       </div>
+      <!-- Change Language Settings -->
+      <div class="place-content-end flex gap-2">
+        <drop-selector :array="i18n.languageOptions" @change="i18n.changeLanguage" class="max-w-40">
```
The language picker should probably be extracted into a component

julianbollig replied:
Done

mschuettlerTNG commented:
While the language picker technically works, the installation management screen texts don't change, because they were not translated.
For demo purposes we should at least translate to German (and maybe Chinese). GPT should make quick work of that.
During the next stakeholder review we can figure out a process for the other translations.

julianbollig commented:
Using TAIA, I updated all language components to German. Also, I added the language mapping to display the current backend status in the corresponding language.
